README.md

```diff
@@ -136,28 +134,17 @@ cd server
 
 cp src/.mf.config.SAMPLE.json src/.mf.config.json
 
+# To install dependencies, clean/create build/ subdirectory, build and upgrade SDK, build source code within this project
 npm install
-npm run clean             # Cleans/creates build/ subdirectory
 
-npm run build:core        # MANDATORY: If you're only using the Firebolt Core SDK (typical for most 3rd party app devs)
-npm run build:manage      # OPTIONAL: If you're using the Firebolt Manage SDK (not typical)
-npm run build:discovery   # OPTIONAL: If you're using the Firebolt Discovery SDK (not typical)
-
-npm run build:mf          # Builds source code within this project
```

Comment:
This is not dependent on the SDKs and should stay.

Reply:
Same as initial comment